### DISCUSSION TITLE
간단한 제페토 축구게임(멀티) 샘플입니다~

### DISCUSSION THREAD

**Discussion by shlifedev** [naverz/zepeto-studio-kor#454] [3. Creator Samples]
간단하게 축구공을 드리블하고 월드에서 놀 수 있는 샘플입니다.  (만든김에 공유..) 
링크에 있는 미구현 사항들에 대해서는 업데이트 될 예정입니다! 

[github link](https://github.com/shlifedev/ZepetoFootball/blob/main/README.md) 에 배포되어 있습니다! 

### 샘플 이미지 
![gif](https://github.com/shlifedev/ZepetoFootball/raw/main/Animation2.gif)

### 샘플 이미지2
![gif](https://github.com/shlifedev/ZepetoFootball/raw/main/Animation.gif)

**Comment by x4cf** [comment]
너무 멋져요!

**Reply by shlifedev** [reply]
감사합니다!! 😊

**Reply by hyunjiko** [reply]
저도 축구하고 싶어요 ㅠㅠ
